app/controllers/provider_interface/organisation_permissions_controller.rb

```diff
@@ -26,6 +26,10 @@ def edit; end
 
     def update
       if @relationship.update(new_relationship_permissions)
+        if FeatureFlag.active?(:accredited_provider_setting_permissions)
```

Comment:
Not sure the feature flag check is required here, since we redirect if the flag is off in the whole of the controller